cmd/agent/subcommands/secret/command.go

```diff
@@ -32,55 +32,78 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 	cliParams := &cliParams{
 		GlobalParams: globalParams,
 	}
+
 	secretInfoCommand := &cobra.Command{
 		Use:   "secret",
-		Short: "Print information about decrypted secrets in configuration.",
+		Short: "Display information secrets in configuration.",
```

Comment:
NIT: should this be:
```suggestion
		Short: "Display information about secrets in configuration.",
```

or am I misunderstanding the context?

Reply:
Nice catch, thanks!